api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -2869,11 +2867,14 @@ private void enableHIVProgram() {
                         "dashboard", programUuid   // provides contextual context so this widget knows which dashboard it's being rendered on
                 )),
                 "coreapps", "program/programHistory",
-                secondColumnIndex++
+                2
         ));
 
         // TODO: Add ACTIVE MEDICATIONS HERE
 
+        // Start index for order after fingerprints
```

Comment:
That's annoying (that you can't just use the "++" because the biometrics widget is added elsewhere).

It took me a few minutes to figure out what was going on here, maybe make this comment "Start index for order after fingerprints (which are added in the biometrics section)"

Reply:
Right @mogoodrich .  I should have explained.  I don't think that we have implementations that don't have the HIV program AND Biometrics, but it's possible.  Figured it was cleaner to include the biometrics dashboard widget with the biometrics component but it makes for this messy ordering of the widgets.

I did test and if the biometrics widget is missing, it still orders widgets properly.